config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = "https://linuxboot.org/"
+baseURL = "http://localhost/"
```

Comment:
this seems strange?

Reply:
Nope. It's normal netlify takes argument for hugo. I changed it because it is easier to use for devs. So the url is passed during static page generation,